app/forms/apply_applications/confirm_course_form.rb

```diff
@@ -65,17 +69,22 @@ def update_trainee_attributes
         course_subject_one: course_subject_one,
         course_subject_two: course_subject_two,
         course_subject_three: course_subject_three,
-        training_route: course&.route,
-        course_code: course.code,
+        training_route: training_route,
```

Comment:
Not sure if this needs to change because it's about copying information from the course to the trainee.

Reply:
reverted